server/odc-service/src/main/java/com/oceanbase/odc/service/task/executor/sampletask/SampleTaskResult.java

```diff
@@ -0,0 +1,44 @@
+/*
+ * Copyright (c) 2023 OceanBase.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.oceanbase.odc.service.task.executor.sampletask;
+
+import com.oceanbase.odc.core.flow.model.FlowTaskResult;
+
+import lombok.Data;
+
+/**
+ * @author gaoda.xy
+ * @date 2023/11/22 20:01
+ */
+@Data
+public class SampleTaskResult implements FlowTaskResult {
```

Comment:
avoid use Flow Class

Reply:
You are right. But we no need to create a new class to represent task executing result. `FlowTaskResult` will be change to `TaskResult` in future.